packages/babel-types/test/converters.js

```diff
@@ -36,6 +36,9 @@ describe("converters", function () {
         ),
       );
     });
+    it("bigint", function () {
+      expect(t.valueToNode(123n)).toEqual(t.bigIntLiteral("123"));
+    });
```

Comment:
This test will fail on Node.js 8 and Node.js 6 because they don't have native bigint support. To make the CI green we can use the `BigInt` constructor and wrap this test with `itGte("10.4.0")` so that the test will only run on Node.js >= 10.4.0. See `eslint/babel-eslint-tests/test/integration/parser-override.js` for an example usage.

Reply:
It looks like doing that caused an eslint error 
> error: Expect must be inside of a test block (jest/no-standalone-expect) at packages/babel-types/test/converters.js:43:7:

What would be the right way to resolve that?

Reply:
Good question, we should add `nodeGte10` to https://github.com/babel/babel/blob/985f051c7c68a09f12ea2089c6cc3d0452b46249/eslint.config.mjs#L271-L291

Reply:
Added that change to this PR as well. In case that needs to be split into a separate PR, let me know.